src/test/java/com/wikia/webdriver/testcases/flagTests/FlagTests.java

```diff
@@ -0,0 +1,50 @@
+package com.wikia.webdriver.testcases.flagtests;
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.flag.AddFlagsComponentObject;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by Mariusz on 2015-06-28.
+ */
+public class FlagTests extends NewTestTemplate {
+
+    Credentials credentials = config.getCredentials();
+
+    @Test(
+            groups = {"addFlag_01"})
+    public void addFlag_01() {
```

Comment:
What is wrong with package naming like this com.wikia.webdriver.testcases.flagtests; ?